### PR TITLE
Add customer role

### PR DESCRIPTION
**Pull request by GeorgiStoyanovAtanasov**

### CODE REVIEW

File: src/main/java/com/example/ludogoriesoft/lukeriaerpapi/dtos/ProductDTO.java [hunk 1]
```diff
@@ -20,4 +20,12 @@ public class ProductDTO {
     private int availableQuantity;
     @Size(min = 2, max = 10, message = "Кода на продукта трябда да бъде между 2 и 10 символа!")
     private String productCode;
+    public ProductDTO(BigDecimal price, Long packageId, int availableQuantity, String productCode) {
+        this.price = price;
+        this.packageId = packageId;
+        this.availableQuantity = availableQuantity;
+        this.productCode = productCode;
+    }
 }
```
Comment (berki0) [inline src/main/java/com/example/ludogoriesoft/lukeriaerpapi/dtos/ProductDTO.java:29]: we have AllArgsConstructor annotation and I think no constructors need to be defined

File: src/main/resources/application.properties
Comment (berki0) [inline src/main/resources/application.properties:1]: please return the pom.xml file and use environment variables

File: src/main/java/com/example/ludogoriesoft/lukeriaerpapi/dtos/ProductDTO.java (hunk 1 above)
Comment (GeorgiStoyanovAtanasov) [reply src/main/java/com/example/ludogoriesoft/lukeriaerpapi/dtos/ProductDTO.java]: the AllArgsConstructor requires id as a param, this one doesn't.

File: src/main/resources/application.properties
Comment (GeorgiStoyanovAtanasov) [reply src/main/resources/application.properties:1]: done

**Pull request merged**